.travis.yml

```
@@ -2,7 +2,11 @@ sudo: required
 language: python
 python:
   - "3.6.4"
-# command to run tests
+services: 
+    - docker
+before_install: 
+    - docker build -t jr/easycrm -f Dockerfile.dev .
 install:
   - pip install -r requirements.txt
-script: nosetests -v
+script:     
+    - docker run -e CI=true jr/easycrm nosetests -v -- --coverage --watchAll=false 
```

```suggestion
  - docker run -e CI=true jr/easycrm nosetests -v -- --coverage --watchAll=false 
```

Same here, need to be care with the format